cli/QuickstepCli.cpp

```diff
@@ -430,6 +445,13 @@ int main(int argc, char* argv[]) {
         reset_parser = true;
         break;
       }
+#ifdef QUICKSTEP_ENABLE_GOOGLE_PROFILER
+      // Profile only if profile_file_name flag is set
+      if (!query_num && !quickstep::FLAGS_profile_file_name.empty()) {
```

Comment:
@navsan Ping again this issue. I did not see we use a bool variable here. Do we go with any alternative options?